### ISSUE TITLE
default.html from create_site easily crashes on compile

### ISSUE THREAD

**Issue by bburton** [nanoc/nanoc#685]
After running the `nanoc create_site` command, the resulting `default.html` layout will crash on compile if the /stylesheet.css file is moved or renamed:

```
Message:

NoMethodError: undefined method `path' for nil:NilClass

Compilation stack:

  - [item]   /index.html (rep default)
```

This is going to cause consternation for a lot of users as almost everyone who creates a new site is going to move or rename stylesheet.css and may not fix the reference in default.html at the same time.

It's really much better to leave the stylesheet.css link hard coded in default.html the way it was since users will quickly figure out they're getting unstyled content and fix the reference.

See https://github.com/nanoc/nanoc/blob/master/lib/nanoc/cli/commands/create-site.rb#L257

**Comment by ddfreyne** [comment]
Same as #684 I believe.

**Comment by ddfreyne** [comment]
Makes sense. Combined with an `internal_links` deploy-time check, the chance of ever deploying a site that is not properly styled is very low.

**Comment by ddfreyne** [comment]
Fixed by #701.
